catalog/task/collect-data/0.1/collect-data.yaml

```diff
@@ -53,32 +53,43 @@ spec:
       script: |
         #!/usr/bin/env sh
```

Comment:
Is there any reason why we don't want `set -e` here? Generally it's preferable to fail whenever something goes wrong if possible, because that makes it easier to debug stuff - rather than fail silently and carry on and then fail later on something and try to figure out why.

That way we wouldn't need the `if`s at all.

Reply:
Because when you asked on the other PR about that, David said he didn't want to, so I assumed he still does not want it

Reply:
I do vaguely recall that, but I can't remember what the reasoning was. But ok.

Reply:
https://github.com/redhat-appstudio/release-service-bundles/pull/135#discussion_r1258383509
I think the `set -e` would be better, but he wrote it originally and seems to have a reason for it so I was trying to stick with not adding it

Reply:
Thanks for finding the comment. So I'm repeating myself when I say that I prefer `set -e` 

@davidmogar do you still insist in using `set +e`? Using `set -e` would make the script much simpler (no need for all the `if`s).

Reply:
@davidmogar ping

Reply:
Cool, thanks.